podcasts/Onboarding/Plus/PlusPurchaseModal.swift

```diff
@@ -141,17 +137,21 @@ private struct PlusDivider: View {
 }
 
 private struct TermsView: View {
-    @State var labelSize: CGSize = .zero
-    let text: String
-
     var body: some View {
-        GeometryReader { geometry in
-            HTMLTextView(text: text,
-                         font: .font(ofSize: 14, weight: .regular, scalingWith: .footnote, maxSizeCategory: .extraExtraLarge),
-                         textColor: Color.uiTextColor,
-                         width: geometry.size.width,
-                         textViewSize: $labelSize)
-        }.frame(height: labelSize.height)
+        let purchaseTerms = L10n.purchaseTerms("$", "$", "$", "$").components(separatedBy: "$")
+
+        let privacyPolicy = ServerConstants.Urls.privacyPolicy
+        let termsOfUse = ServerConstants.Urls.termsOfUse
+
+        Group {
+            Text(purchaseTerms[safe: 0] ?? "") +
+            Text(.init("[\(purchaseTerms[safe: 1] ?? "")](\(privacyPolicy))")).underline() +
+            Text(purchaseTerms[safe: 2] ?? "") +
+            Text(.init("[\(purchaseTerms[safe: 3] ?? "")](\(termsOfUse))")).underline()
+        }
+        .foregroundColor(.textColor)
+        .font(style: .footnote)
+        .fixedSize(horizontal: false, vertical: true)
```

Comment:
A few things here:
- [`foregroundColor`](https://developer.apple.com/documentation/SwiftUI/View/foregroundColor(_:)) is being deprecated so we should use `foregroundStyle` instead
- The link color is using the default color, we can change this by using `.tint(.textColor)`.
- I noticed the font size was slightly smaller than the designs and should be centered not leading aligned. 

```suggestion
        .multilineTextAlignment(.center)
        .foregroundStyle(Color.textColor)
        .tint(.textColor)
        .font(size: 14, style: .subheadline)
        .fixedSize(horizontal: false, vertical: true)
```